src/utils/formatters.js

```diff
@@ -1,6 +1,9 @@
 import { get, round, find, some, map } from 'lodash-es';
 import { format } from 'date-fns';
 
+export const isNull = inputArr =>
+  inputArr.join().replace(/,/g, '').length === 0;
```

Comment:
why this instead of a simple check for empty strings or null values?

Reply:
No real reason. I just felt like being an idiot. Fixed?